nixos/modules/virtualisation/oci-containers.nix

```
@@ -221,6 +221,12 @@ let
           example = "hello-world";
         };
 
+        preRunExtraOptions = mkOption {
+          type = with types; listOf str;
+          default = [];
+          description = "Extra options for {command}`${defaultBackend}`.";
```

Could you provide an example of what someone might use this for?  And perhaps clarify the description a bit, maybe something like...

```suggestion
          description = "Extra options for {command}`${defaultBackend}` that go before the `run` argument.";
          example = ["--log-level=info"];
```